cli/src/main/java/com/devonfw/tools/ide/tool/mvn/Mvn.java

```diff
@@ -185,8 +185,9 @@ private Set<String> findVariables(String content) {
   @Override
   public void installPlugin(PluginDescriptor plugin) {
 
-    Path mavenPlugin = this.getToolPath().resolve("lib/ext/" + plugin.getName() + ".jar");
+    Path mavenPlugin = this.context.getSoftwarePath().resolve(this.tool).resolve("lib/ext/" + plugin.getName() + ".jar");
```

Comment:
Not wrong but just for my curiosity: Why did you had to change this? The method `getToolPath()` should give us the same result and makes it more compact and readable IMHO. I assume there is a reason why you changed this that I am missing.

Reply:
This looks like an unintended change. Should be fixed with: #595